web/js/FlexyConnector.js

```diff
@@ -26,7 +26,8 @@ function saveJSON()
    //Fetch the AppConfig
    var CycleTimeMs = document.getElementById("CycleTimeMs").value;
    var PostDuplicateTagValues = document.getElementById("PostDuplicateTagValues").value;
-   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues };
+   var OMF = document.getElementById("CommunicationType").value;
+   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues, communicationType: communicationType };

```

Comment:
CommunicationType needs to be capital

Reply:
yeah the variable is still called OMF as well.. Fixed